src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,689 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Grid, Row } from '../../components/grid'
+import BlueBackground from '../../components/Common/BlueBackground'
+import GreyBackground from '../../components/Common/GreyBackground'
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import {
+  TextColumnsBlock,
+  TextAndImageBlock,
+  FullWidthBlock,
+  VideoBlock,
+  ImagesBlock
+} from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+
+// import SubtitleWithBody from '../../components/Common/SubtitleWithBody'
+// import Statement from '../../components/Common/Statement'
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+
+const BlockRow = styled(Row)`
+  flex-direction: ${({ rowReverse }) => `row${rowReverse ? '-reverse' : null}`};
+  align-items: ${({ alignCenter }) => (alignCenter ? 'center' : null)};
+  padding-top: ${({ theme, mobile }) =>
+    mobile && mobile.top ? theme.space[mobile.top] : null};
+  padding-bottom: ${({ theme, mobile }) =>
+    mobile && mobile.bottom ? theme.space[mobile.bottom] : null};
+  width: ${({ widthPerc }) => widthPerc};
+  justify-content: ${({ spaceEvenly }) =>
+    spaceEvenly ? 'space-evenly' : null};
+
+${breakpoint('smallTablet')`
+  padding-top: ${({ theme, smallTablet }) =>
+    smallTablet && smallTablet.top ? theme.space[smallTablet.top] : null};
+  padding-bottom: ${({ theme, smallTablet }) =>
+    smallTablet && smallTablet.bottom ? theme.space[smallTablet.bottom] : null};
+`}
+
+${breakpoint('tablet')`
+  padding-top: ${({ theme, tablet }) =>
+    tablet && tablet.top ? theme.space[tablet.top] : null};
+  padding-bottom: ${({ theme, tablet }) =>
+    tablet && tablet.bottom ? theme.space[tablet.bottom] : null};
+`}
+
+${breakpoint('desktop')`
+  padding-top: ${({ theme, desktop }) =>
+    desktop && desktop.top ? theme.space[desktop.top] : null};
+  padding-bottom: ${({ theme, desktop }) =>
+    desktop && desktop.bottom ? theme.space[desktop.bottom] : null};
+`}
+`
+
+const ReversableDiv = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+
+  ${breakpoint('smallTablet')`
+    flex-direction: column;
+  `}
+`
+
+const addPaddings = (top, bottom) => ({
+  top,
+  bottom
+})
+
+const getImage = (blockImages, index) => blockImages && blockImages[index]
+
+const normalise = (arr = []) => {
+  const normalised = arr.map(
+    ({ genericBlockText, genericBlockImages, ...props }) => ({
+      image: getImage(genericBlockImages, 0),
+      text: genericBlockText && genericBlockText.genericBlockText,
+      ...props
+    })
+  )
+  return normalised[0]
+}
+
+const normaliseImages = ({ genericBlockImages = [] }) => genericBlockImages
```

Comment:
This is a bit confusing to understand at first. If you want to write a `get` function with a default I would make it a little clearer. At first I thought it was just returning itself.

```
const getImages = (data) => data.genericBlockImages ? data.genericBlockImages  : []
```

Reply:
Good idea. Will do.

Reply:
Done.